src/components/Request/Pay/Views/Initial.view.tsx

```diff
@@ -55,7 +60,10 @@ export const InitialView = ({
     useEffect(() => {
         const estimateTxFee = async () => {
             setLinkState(RequestStatus.LOADING)
-            if (selectedChainID === requestLinkData.chainId && selectedTokenAddress === requestLinkData.tokenAddress) {
+            if (
+                selectedChainID === requestLinkData.chainId
+                && utils.compareTokenAddresses(selectedTokenAddress, requestLinkData.tokenAddress)
```

Comment:
thought: this should prob be called `areTokenAddressesEqual` or equivalent or `isSameTokenAddress`

Reply:
I agree, renamed